e2e/fixtures/schedule.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  allocation: 'The proceedings continue to be allocated to Lewis Birks',
```

Comment:
Is Lewis happy having his real name in the test data? :P

Reply:
I'm fine with it

Reply:
Name changed :)